### PR TITLE
Update Natural and artificial perturbations with combined perturbations

### PR DESCRIPTION
**Pull request by WolfsSky**
Please re-run the notebook first, not sure if I did it correctly! (first time here)

### CODE REVIEW

File: docs/source/examples/Natural and artificial perturbations.ipynb [hunk 1]
```diff
@@ -280,32 +280,122 @@
...
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 14,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "ename": "NameError",
```
Comment (Juanlu001) [inline docs/source/examples/Natural and artificial perturbations.ipynb]: Oops, this is an error right?

Comment (WolfsSky) [reply docs/source/examples/Natural and artificial perturbations.ipynb]: Will take a look tomorrow!

File: docs/source/examples/Natural and artificial perturbations.ipynb [hunk 2]
```diff
@@ -11,7 +11,16 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "gzip was not found on your system! You should solve this issue for astroquery.eso to be at its best!\n",
```
Comment (Juanlu001) [inline docs/source/examples/Natural and artificial perturbations.ipynb]: Interesting, I was not aware of this issue on Windows. Let's leave it like that anyway.

**Comment by WolfsSky** [comment]
Thank you too! I saw those as well, those fail on my PC as well, they are perturbations related I think.

File: docs/source/examples/Natural and artificial perturbations.ipynb (hunk 2 above)
Comment (WolfsSky) [reply docs/source/examples/Natural and artificial perturbations.ipynb]: Yes I installed 7zip, but the error still shows.

**Comment by WolfsSky** [comment]
I just see two typos now: 
1. Mulriple = Multiple, in the new title
2. one of the comments has atmospehric instead of atmospheric

**Comment by Juanlu001** [comment]
Regarding the failing tests, I updated them here:

https://github.com/poliastro/poliastro/pull/692

**Comment by Juanlu001** [comment]
@WolfsSky I rebased your pull request on top of current master to make the corrected tests run. Will merge after they pass!

**Comment by Juanlu001** [comment]
For the record, I did

```
$ git checkout master
$ git fetch upstream  # Assuming upstream points to https://github.com/poliastro/poliastro/
$ git rebase upstream/master
```

Didn't want to put this burden on your first pull request, git can be hard :)

File: docs/source/examples/Natural and artificial perturbations.ipynb [hunk 3]
```diff
@@ -97,7 +106,7 @@
    "source": [
     "plt.ylabel(\"h(t)\")\n",
     "plt.xlabel(\"t, days\")\n",
-    "plt.plot(tofs.value, rr.norm() - Earth.R);"
+    "plt.plot(tofs.value, rr.data.norm() - Earth.R);"
```
Comment (Juanlu001) [inline docs/source/examples/Natural and artificial perturbations.ipynb]: Oops, I didn't notice these ones. You were probably using the latest stable version, but it changed in the development version.

**Comment by Juanlu001** [comment]
```
Traceback (most recent call last)
<ipython-input-3-10bb1889b875> in <module>
      1 plt.ylabel("h(t)")
      2 plt.xlabel("t, days")
----> 3 plt.plot(tofs.value, rr.data.norm() - Earth.R);

AttributeError: 'CartesianRepresentation' object has no attribute 'data'
AttributeError: 'CartesianRepresentation' object has no attribute 'data'
```

https://circleci.com/gh/poliastro/poliastro/4127

**Comment by Juanlu001** [comment]
Hope you don't mind I did some final adjustments myself. Will merge when the tests pass!

**Comment by WolfsSky** [comment]
No worries, I was a bit confused by the different versions. And indeed I do use the stable version. Will keep this in mind for next time.

**Comment by Juanlu001** [comment]
Yes, poliastro is still a bit unstable so we make little breaking changes between releases. Hopefully we add enough awesome new features to motivate users to upgrade :wink: In 2020 we might be in a position to release poliastro 1.0 and be a bit more strict with backwards compatibility.

**Pull request merged**